### PR TITLE
Release 1.2

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

Preparing release 1.2

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-# DGtal 1.2 (dev)
+# DGtal 1.2
 
 ## New Features / Critical Changes
 
+- *New Feature*
+  - DGtal now has a python binding `pip install dgtal`! For all
+    details on the list of classes available in python, you can have a
+    loo to: Pablo Hernandez-Cerdan [#1543](https://github.com/DGtal-team/DGtal/pull/1528)
```
Comment (phcerdan) [inline ChangeLog.md:8]: loo -> look

I think is good to link to the PR, but there is also a README, with the same information, in the `wrap` folder here: https://github.com/DGtal-team/DGtal/tree/master/wrap

Comment (dcoeurjo) [inline ChangeLog.md:8]:
```suggestion
    look to: Pablo Hernandez-Cerdan [#1543](https://github.com/DGtal-team/DGtal/pull/1528) 
```

Comment (dcoeurjo) [reply ChangeLog.md:8]: thanks for the feedback

**Comment by dcoeurjo** [comment]
I would need a review for this one, @kerautret @JacquesOlivierLachaud

File: ChangeLog.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
-# DGtal 1.2 (dev)
+# DGtal 1.2
 
 ## New Features / Critical Changes
 
+- *New Feature*
+  - DGtal now has a python binding `pip install dgtal`! For all
+    details on the list of classes available in python, you can have a
+    look to: Pablo Hernandez-Cerdan [#1543](https://github.com/DGtal-team/DGtal/pull/1528)
```
Comment (kerautret) [inline ChangeLog.md:8]:
```suggestion
    look to: Pablo Hernandez-Cerdan [#1528](https://github.com/DGtal-team/DGtal/pull/1528) 
```

**Review by kerautret — APPROVED** [review]

File: ChangeLog.md (hunk 2 above)
Comment (dcoeurjo) [reply ChangeLog.md:8]: oops thx

**Comment by kerautret** [comment]
done @dcoeurjo all nice just a typo on PR number ! By the way good idea the example in the doc just tested successfully in few minutes ! 👍🏻👍🏻👍🏻

**Pull request merged**